shared/pages/import/js/oauth20.js

```diff
@@ -47,7 +47,7 @@ if (typeof window.oauth2 === 'undefined') {
 
       // Enables simple access to test tokens for hacking up the app
       // This code will need to be deleted once we have a final product
-      fb.testToken = fb.testToken || parent.fb.testToken;
+      fb.testToken = fb.testToken || (parent.fb && parent.fb.testToken);
```

Comment:
yeah, it triggers a null parent error (fb is used by gmail, don't ask me why), so we better check that parent exists first. 
other option would be to fully refactor the fb.token, but I'd leave that for future facebook related bugs (too complex for doing it here, where we just want to hide the toggle)